src/services/account.service.ts

```diff
@@ -4,11 +4,10 @@ import { APIService } from './api-service';
 
 export class AccountService extends APIService {
   getAccount = async (userAccessToken: AccessToken): Promise<APIResponse> => {
-    return this.get(`/accounts/${userAccessToken.accountId}`, {
-      headers: {
-        Authorization: `Bearer ${userAccessToken.token}`,
-      },
-    });
+    return this.get(
+      `/accounts/${userAccessToken.accountId}`,
+      this.getBearerTokenHeader(userAccessToken.token),
```

Comment:
Previous name was okay - `getAuthorizationHeader`. `Bearer` is an internal implementation detail that caller does not needs to be aware about.

Reply:
Okay, got it.